packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -446,8 +589,39 @@ export default {
     this.analyticsValues = getAnalyticsValues();
   },
   methods: {
+    setHeight(isNotUSAAndCanada) {
```

Comment:
Can you explain the need for this function? Why do we need to resize the form when submitting?

Reply:
I'm calling this function here because if the user tries to submit an empty form and they get errors next to the fields, then the popup's height needs to be recalculated.